src/NLog/Internal/ExceptionHelper.cs

```diff
@@ -63,17 +155,35 @@ public static bool MustBeRethrown(this Exception exception)
                 return true;
             }
 
-            if (exception is NLogConfigurationException)
+            // only log after the 'serious' exceptions above
+
+            var shallRethrow = false;
+
+            if (!shallRethrow)
```

Comment:
proposal: remove unneeded `if` and combine asignment & declaration of "shallRethrow"